### PR TITLE
Add `meroxa functions`

### PR DESCRIPTION
**Pull request by owenthereal**
# Description of change

Add `meroxa functions`. This depends on https://github.com/meroxa/meroxa-go/pull/90.

# Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

# How was this tested?

- [x]  Unit Tests
- [x]  Tested in staging

# Demo

```
$ m function create o-func --input-stream resource-4497-028165-users --image meroxa/funtime-example-jq --pipeline default --env JQ_QUERY=.
Function "o-func" successfully created!

$ m function list
                 UUID                    NAME            INPUT STREAM                    OUTPUT STREAM               STATE    PIPELINE
====================================== ========= ============================ ==================================== ========= ==========
 85e18535-1505-4fe4-97cb-da209687b597   o-func    resource-4497-028165-users   resource-4497-028165-users-o-func    running   default

$ m function describe o-func
          UUID:   85e18535-1505-4fe4-97cb-da209687b597
          Name:   o-func
  Input Stream:   resource-4497-028165-users
 Output Stream:   resource-4497-028165-users-o-func
         Image:   meroxa/funtime-example-jq
       Command:
     Arguments:
      Env Vars:   JQ_QUERY=.
      Pipeline:   default
         State:   Running

$ m function delete o-func
To proceed, type "o-func" or re-run this command with --force
▸ o-func
Function "o-func" is being removed...
Function "o-func" successfully removed
```

### CODE REVIEW

**Review by ahmeroxa — APPROVED** [review]
LGTM 👍

**Comment by owenthereal** [comment]
The ftest test failed for `--- FAIL: TestSQLServerToSQLServer/cleaning_up_create_test_db_testdb_vklu8e (25.80s)` which is not relevant to this PR. I'm going to merge this to unblock other items.

File: cmd/meroxa/root/functions/functions.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+func (*Functions) Docs() builder.Docs {
+	return builder.Docs{
+		Short: "Manage functions on Meroxa",
+	}
+}
+
+func (*Functions) Aliases() []string {
+	return []string{"function"}
```
Comment (raulb) [inline cmd/meroxa/root/functions/functions.go:39]: it'd be nice to maybe also have `fn` as an alias? Just a thought.

**Review by raulb — COMMENTED** [review]
🚀

**Pull request merged**